index.html

```diff
@@ -56,6 +56,27 @@
         <img src="heart.svg" title="♥" style="max-width: clamp(1rem, 4rem + 10vw, 20rem); max-height: 20vh;">
         <h1 style="margin-block: 0;">Act Different.</h1>
       </hgroup>
+      <p>We, developers of software for your platform, ask that you begin to change course.</p>
```

Comment:
That's merely formulaic and needs to go.

How do we want to open this? With a plea? With a vision?